pkg/scheduler/core/generic_scheduler.go

```diff
@@ -360,13 +369,27 @@ func (g *genericScheduler) findNodesThatPassExtenders(pod *v1.Pod, feasibleNodes
 			return nil, err
 		}
 
+		for failedNodeName, failedMsg := range failedAndUnresolvableMap {
+			var aggregatedReasons []string
+			if _, found := statuses[failedNodeName]; found {
+				aggregatedReasons = statuses[failedNodeName].Reasons()
+			}
+			aggregatedReasons = append(aggregatedReasons, failedMsg)
+			statuses[failedNodeName] = framework.NewStatus(framework.UnschedulableAndUnresolvable, aggregatedReasons...)
+		}
+
 		for failedNodeName, failedMsg := range failedMap {
+			if _, found := failedAndUnresolvableMap[failedNodeName]; found {
```

Comment:
would this happen?
I can only think of the case where the extender returns both states for the same Node.

Reply:
yes, ignore the nodes in `failedMap` if they are already in `failedAndUnresolvableMap`

Reply:
please add a comment that this only happens if the extender returns it in both

Reply:
added, PTAL